manager/perf.go

```diff
@@ -107,20 +151,24 @@ func (m *PerfMap) Start() error {
 			record, err = m.perfReader.Read()
 			if err != nil {
 				if perf.IsClosed(err) {
+					m.PerfMapStats.Closed++
 					m.manager.wg.Done()
 					return
 				}
+				m.PerfMapStats.ReadErrors++
 				if m.PerfErrChan != nil {
 					m.PerfErrChan <- err
 				}
 				continue
 			}
 			if record.LostSamples > 0 {
+				m.PerfMapStats.LostSamples[record.CPU] += record.LostSamples
 				if m.LostHandler != nil {
 					m.LostHandler(record.CPU, record.LostSamples, m, m.manager)
 				}
 				continue
 			}
+			m.PerfMapStats.RawSamples[record.CPU] += uint64(len(record.RawSample))
```

Comment:
Could we have a way to opt into statkeeping? Perf maps are often on a hot path, and thus I wouldn't want to introduce unnecessary overhead.

Reply:
I would not count a this.map[int]int as costly (hashkey is easy and mostly value update) in the hot path but, we can disable PerfMapStats.

I did via pointer, so we can enable stats runtime and avoid using unnecessary memory